### PR TITLE
Reporters trigger 2

### PR DESCRIPTION
**Pull request by norkans7**
Should be merged after #81 is pushed

### CODE REVIEW

**Comment by norkans7** [comment]
This is ready, @nicpottier please review

File: ureport/utils/tests.py [hunk 1]
```diff
@@ -543,3 +547,136 @@ def test_fetch_old_sites_count(self):
                                                        UREPORT_ASYNC_FETCHED_DATA_CACHE_TIME)
 
                         cache_delete_mock.assert_called_once_with(GLOBAL_COUNT_CACHE_KEY)
+
+    def test_get_gender_stats(self):
+
+        self.assertEqual(get_gender_stats(self.org), dict(female_count=0, female_percentage="---",
+                                                          male_count=0, male_percentage="---"))
+
+        ReportersCounter.objects.create(org=self.org, type='gender:f', count=2)
```
Comment (nicpottier) [inline ureport/utils/tests.py]: Maybe add another row to test the squashed vs unsquashed state.

File: ureport/utils/tests.py [hunk 2]
```diff
@@ -543,3 +547,136 @@ def test_fetch_old_sites_count(self):
...
+        with patch.object(timezone, 'now', return_value=tz.localize(datetime(2015, 9, 4, 3, 4, 5, 6))):
+
+            stats = json.loads(get_registration_stats(self.org))
+
+            for entry in stats:
+                self.assertEqual(entry['count'], 0)
+
+            ReportersCounter.objects.create(org=self.org, type='registered_on:2015-08-27', count=3)
+            ReportersCounter.objects.create(org=self.org, type='registered_on:2015-08-25', count=6)
+            ReportersCounter.objects.create(org=self.org, type='registered_on:2015-06-30', count=4)
```
Comment (nicpottier) [inline ureport/utils/tests.py]: Same here, having some repeats is more realistic.

**Comment by nicpottier** [comment]
Tweak those tests to be a bit more comprehensive but looks sane.

**Pull request merged**